pkg/capture/capturetypes/config.go

```diff
@@ -0,0 +1,41 @@
+package capturetypes
+
+// IfaceChange denotes the result from an interface / config update of an interface
+type IfaceChange struct {
+	Name    string
```

Comment:
Please add comments so it's easier for ChatGPT to churn out an OpenAPI spec.
```suggestion
	Name    string // Name: the name of the interface. Example: "eth0"
	Success bool // Success: indicates whether the operation succeeded. Example: true
```

Reply:
Sure thing. Will take care of the OpenAPI extensions , too...

Reply:
Oh! That's on me then.

I guess shared pain is half the pain then 🤓🤪

Reply:
Ah, never mind, I was just looking in the wrong place. Base path was already there, I amended the changes to the config update call.